app/src/main/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentPresenter.kt

```diff
@@ -180,28 +184,42 @@ class TopicLessonsFragmentPresenter @Inject constructor(
     )
     binding.chapterRecyclerView.adapter = createChapterRecyclerViewAdapter()
 
+    binding.chapterListDropDownIcon.setOnClickListener {
+      expandStoryList(position)
+    }
+
     binding.root.setOnClickListener {
-      val previousIndex: Int? = currentExpandedChapterListIndex
-      currentExpandedChapterListIndex =
-        if (currentExpandedChapterListIndex != null &&
-          currentExpandedChapterListIndex == position
-        ) {
-          null
-        } else {
-          position
-        }
-      expandedChapterListIndexListener.onExpandListIconClicked(currentExpandedChapterListIndex)
-      if (previousIndex != null && currentExpandedChapterListIndex != null &&
-        previousIndex == currentExpandedChapterListIndex
+      expandStoryList(position)
+    }
+
+    if (accessibilityService.isScreenReaderEnabled()) {
+      binding.root.isClickable = false
+    } else {
+      binding.chapterListDropDownIcon.isClickable = false
+    }
```

Comment:
This logic is incorrect.

Maybe this can solve the problem (not sure)

Try this:
1. Keep the app **active**, enable the any A11Y service and now disable it and again disable it
2. Notice the entire expand click functionality is fully disabled.

```
    if (accessibilityService.isScreenReaderEnabled()) {
      binding.root.isClickable = false
      binding.chapterListDropDownIcon.isClickable = true
    } else {
      binding.root.isClickable = true
      binding.chapterListDropDownIcon.isClickable = false
    }
```

Reply:
Updated according to your suggestion

Reply:
As per the other comment, if you are using `expandListIcon` then we need to make `expandListIcon ` clickable/not-clickable

@vrajdesai78 Please check the PR throughly and also CI checks before submitting. You might need to update some of previous test cases.